tests/integration-tests/tests/performance_tests/test_openfoam.py

```diff
@@ -9,7 +9,7 @@
 # builds and installs many things
 TASK_VCPUS = 36  # vCPUs are cut in a half because multithreading is disabled
 BASELINE_CLUSTER_SIZE_ELAPSED_SECONDS = {8: 742, 16: 376, 32: 185}
-PERF_TEST_DIFFERENCE_TOLERANCE = 5
+PERF_TEST_DIFFERENCE_TOLERANCE = 3
```

Comment:
Why do we decrease the tolerance?

Reply:
Because 3% is the suggested tolerance according to our internal consultations.